deflate_medium.c

```diff
@@ -182,7 +182,7 @@ ZLIB_INTERNAL block_state deflate_medium(deflate_state *s, int flush) {
             if (s->lookahead < MIN_LOOKAHEAD && flush == Z_NO_FLUSH) {
```

Comment:
Why not `UNLIKELY` here too?

Reply:
I am not sure, but benchmark-wise it hurt performance.

Reply:
Ok sounds good